local-cli/templates/HelloWorld/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,11 +1,18 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     package="com.helloworld"
     android:versionCode="1"
     android:versionName="1.0">
 
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
 
+    <!-- These permissions are added implicitly by android-jsc, see https://github.com/facebook/android-jsc/pull/12 -->
+    <!-- The following lines can be removed after android-jsc is updated in react-native -->
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" tools:node="remove" />
```

Comment:
I think we should make it clear here that this does not add the permissions and what needs to be done if someone wants to add those to their app (probably just remove `tools:node="remove"`).